internal/platform/database/migrations.go

```
@@ -13,36 +14,275 @@ import (
+	var actualIndexType string
+	var actualNonUnique int
+	var actualIgnored string
+	for rows.Next() {
+		var column string
+		var subPart sql.NullInt64
+		var collation sql.NullString
+		var indexType string
+		var nonUnique int
+		var ignored string
+		if err := rows.Scan(&column, &subPart, &collation, &indexType, &nonUnique, &ignored); err != nil {
+			rows.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `rows.Close` is not checked

(errcheck)